apps/api/.env.example

```diff
@@ -13,7 +13,10 @@ ALGOD_PORT=
 ALGOD_TOKEN=
 ALGOD_ENV=testnet
 
-# CMS configuration (CMS_ACCESS_TOKEN should match the KEY environment variable in the CMS .env)
+### CMS configuration
+# CMS_ACCESS_TOKEN should match...
+# - the KEY environment variable in the CMS .env
+# - the Admin User > Admin Options > Token field in the CMS UI
```

Comment:
Sorry, this is incorrect (the old one was wrong too). This should **not** match the `KEY` environment variable from the CMS. That is just used for the CMS session cookie I believe. `CMS_ACCESS_TOKEN` should **only** match the latter (Admin User > Admin Options > Token field in the CMS UI).

Reply:
This process was documented in the some of the main and CMS READMEs. I moved it to the API README, as that seemed like the right place for it.